common/src/main/res/values/strings.xml

```diff
@@ -1054,9 +1054,8 @@
     <string name="widget_todo_refresh">Refresh to-do</string>
     <string name="widget_todo_label">To-do list</string>
     <string name="widget_todo_empty">You have no to-do items!</string>
+    <string name="widget_todo_no_configuration">Oops. It looks like this widget is not configured.</string>
```

Comment:
This string is more generic, so maybe it's a good idea to make the name more generic as well.